odata-json-format/7 Structural Property.md

```diff
@@ -183,7 +183,8 @@ collections.
 
 The value of a property of type `Collection(Edm.Untyped)`MUST
 be a collection, and it MAY contain any combination of primitive values,
-structural values, and collections.
+structural values, and collections. Enum values within an untyped 
+collection SHOULD be represented as a string, using the member name.
```

Comment:
```suggestion
structural values, and collections. Enumeration values within an untyped 
collection SHOULD be represented as a string, using the `enumerationMember`.
```
Use the same wording as in section 7.1.

Reply:
In other specs we only use the long form "enumeration value".

Reply:
fixed.